package.json

```diff
@@ -4,7 +4,8 @@
   "main": "src/index.js",
   "scripts": {
     "start": "node .",
```

Comment:
With typescript, I think this command it's not working
Your index.js file is in build folder according to tsconfig.json